doc/man/bitcoin-util.1

```diff
@@ -0,0 +1,5 @@
+.TH BITCOIN-UTIL "1"
+.SH NAME
+bitcoin-util \- manual page for bitcoin-util
+
+This is a placefolder file. Please follow the instructions in \fIcontrib/devtools/README.md\fR to generate the manual pages after a release.
```

Comment:
```suggestion
This is a placeholder file. Please follow the instructions in \fIcontrib/devtools/README.md\fR to generate the manual pages after a release.
```

Reply:
Thanks. We'll want to change this in the other files as well.